File: Modul-5/PRAK504/PRAK504-2310817210010-Harry-Pratama-Yunus.c
#include <stdio.h>
#include <string.h>
int reverse(int a){
    int A=0,b;
    for(int i=a;i>0;i/=10){
        b=i%10;
        A=A*10+b;
    }
return A;
}

int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C = A+B;
    printf("%d",reverse(C));
}

File: Modul-5/PRAK504/PRAK504-2310817210010-Harry-Pratama-Yunus.py
def reverse(a):
    A=0
    while a>0:
        b=a%10
        A=A*10+b
        a//=10
    return A

A,B=map(int,input().split())
A=reverse(A)
B=reverse(B)
C = A+B
print(reverse(C))